microk8s-resources/wrappers/microk8s-start.wrapper

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+
+export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
+
+sudo snap start ${SNAP_NAME} --enable
```

Comment:
Can we check the return code here and potentially offer help?  Something like,
```bash
if rc is 0
  if lock exists
    remove lock
  fi
else
  ## maybe pkill or otherwise stop any half-started services?
  echo 'Failed to start microk8s. See snap logs or journalctl or ...'
  exit 1
fi
```
I know we're `set -e`, so a failed `snap start` would exit before the lock removal, but I think self-documenting would be useful here.